### ISSUE TITLE
Spooner Feature Request: Ped Conversations

### ISSUE THREAD

**Issue by ontickjayden** [itsjustcurtis/MenyooSP#176]
Hi, I'd like to request a task sequence for menyoo spooner where I can make peds can have conversations like normal npcs do in GTA. i don't know if this already exists but the closest I can find is play speech which just repeats the same 3 lines

**Comment by itsjustcurtis** [comment]
I don't believe this is possible with natives; I can look into it but I imagine I'll need to close this as wontfix after I've done some digging.
